kbi/0019/index.rst

```diff
@@ -0,0 +1,35 @@
+.. index::
+   single: credentials;interactive authentication prompt
+
+KBI0019: Workaround to an absent interactive authentication prompt
+==================================================================
+
+:authors: Stephan Heunis <jsheunis@gmail.com>
+:discussion: <link>
```

Comment:
```suggestion
:discussion: https://github.com/psychoinformatics-de/knowledge-base/pull/79
```

I'm never sure whether we should link issues or PRs - here there's more discussion in the issue, but other KBIs link to PRs - and since a PR is always linked to an issue it's probably better anyway...